src/utils/storage.ts

```diff
@@ -403,6 +403,15 @@ async function updateTokensData(storage: IStorage, tokens: Set<string>): Promise
   > {
     let retryCount = 0;
 
+    if (uid === NATIVE_TOKEN_UID) {
```

Comment:
Why do you need this now? Was it a bug?

Reply:
No, its just that before the storage would always start with HTR but this changed and if the wallet does not add the native token it can reach a state where we ask the fullnode for the native token details.
This is just to save the call to the fullnode by using the configured native token.